src/mudlet.cpp

```diff
@@ -2373,6 +2401,8 @@ void mudlet::assignKeySequences()
         reconnectShortcut = new QShortcut(reconnectKeySequence, this);
         connect(reconnectShortcut.data(), &QShortcut::activated, this, &mudlet::slot_reconnect);
         dactionReconnect->setShortcut(QKeySequence());
+
+        dactionCloseProfile->setShortcut(QKeySequence());
```

Comment:
You need extra stuff to accomplish the same effect as the others. (Part 1 of 2):
```suggestion
        delete closeShortcut.data();
        closeShortcut = new QShortcut(closeProfileKeySequence, this);
        connect(closeShortcut.data(), &QShortcut::activated, this, &mudlet::slot_close_current_profile);
        dactionCloseProfile->setShortcut(QKeySequence());
```

Reply:
Fixed